rules/event-scopes.js

```diff
@@ -0,0 +1,120 @@
+const {
+  is,
+  isAny
+} = require('bpmnlint-utils');
+
+const {
+  getEventDefinition,
+} = require('./helper');
+
+module.exports = function() {
+  function check(node, reporter) {
+    if (!isAny(node, [
+      'bpmn:Activity',
+      'bpmn:FlowElementsContainer'
+    ])) {
+      return;
+    }
+
+    const catchAllErrorEvents = [],
+          catchAllEscalationEvents = [];
+
+    // check process, sub-process
+    if (is(node, 'bpmn:FlowElementsContainer')) {
+      node.get('flowElements').forEach(flowElement => {
+        if (is(flowElement, 'bpmn:SubProcess') && flowElement.get('triggeredByEvent')) {
+          flowElement.get('flowElements').forEach(eventSubProcessFlowElement => {
+            if (isCatchAllErrorEvent(eventSubProcessFlowElement)) {
+              catchAllErrorEvents.push(eventSubProcessFlowElement);
+            } else if (isCatchAllEscalationEvent(eventSubProcessFlowElement)) {
+              catchAllEscalationEvents.push(eventSubProcessFlowElement);
+            }
+          });
+        }
+      });
+    }
+
+    // check call activity, sub-process, task
+    if (is(node, 'bpmn:Activity')) {
+      const parentNode = node.$parent;
+
+      if (!parentNode) {
+        return;
+      }
+
+      parentNode.get('flowElements').forEach(flowElement => {
+        if (flowElement.get('attachedToRef') === node) {
+          if (isCatchAllErrorEvent(flowElement)) {
+            catchAllErrorEvents.push(flowElement);
+          } else if (isCatchAllEscalationEvent(flowElement)) {
+            catchAllEscalationEvents.push(flowElement);
+          }
+        }
+      });
+    }
+
+    if (catchAllErrorEvents.length > 1) {
+      catchAllErrorEvents.forEach(catchAllErrorEvent => {
+        reporter.report(
+          catchAllErrorEvent.get('id'),
+          'More than one error event without error code in scope'
```

Comment:
Consider if we rather make this a generic error (`Interrupting event subscription conflicts with "XYZ" in same scope`).